pkg/machines/components/libvirtSlate.jsx

```diff
@@ -76,6 +77,9 @@ class LibvirtSlate extends React.Component {
         } else if (name && activeState === 'unknown') { // name === 'unknown' first
             message = _("Connecting to Virtualization Service");
             icon = (<div className="spinner spinner-lg" />);
+        } else if (loadingResources) {
```

Comment:
I think it's already better than i was so I will not block on it.
But I would suggest to not make the first message dissapear when we show the 'Loading Resouces' one, rather we could show the progress by showing both.